src/apps/reservation-list/modal/reservation-details/physical-list-details.tsx

```diff
@@ -41,13 +41,32 @@ const PhysicalListDetails: FC<PhysicalListDetailsProps & MaterialProps> = ({
   branches
 }) => {
   const t = useText();
+
+  const {
+    numberInQueue,
+    pickupBranch,
+    expiryDate: reservationExpiryDate,
+    pickupDeadline,
+    dateOfReservation,
+    pickupNumber,
+    reservationId
+  } = reservation;
```

Comment:
You can also destruct all these value from the reservation in the constructor of the function:
```typescript
const PhysicalListDetails: FC<PhysicalListDetailsProps & MaterialProps> = ({
  reservation: {
    numberInQueue,
    pickupBranch,
    expiryDate: reservationExpiryDate,
    pickupDeadline,
    dateOfReservation,
    pickupNumber,
    reservationId
  },
  branches
}) => {